1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,27 @@
 function readingList(books) {
-  // Write your code here...
+ 
+const divEl = document.getElementById("content")
```

Comment:
```suggestion
  const divEl = document.getElementById("content")
```
These variable declarations should be indented - remember to use Prettier or another code formatter in order to catch these formatting issues _before_ you commit your code